src/registrar/fixtures_domain_requests.py

```diff
@@ -3,13 +3,7 @@
 from faker import Faker
 from django.db import transaction
 
-from registrar.models import (
-    User,
-    DomainRequest,
-    DraftDomain,
-    Contact,
-    Website,
-)
+from registrar.models import User, DomainRequest, DraftDomain, Contact, Website, FederalAgency
```

Comment:
(nonblocking) Was this a linter thing?  Kindof favored the more readable layout.  Personal opinion, really.  Some coders prefer compact.

Reply:
I don't recall us editing this, so I believe it was a linter thing!